pkg/codegen/internal/test/testdata/simple-enum-schema/go/plant/tree/v1/rubberTree.go

```diff
@@ -64,19 +64,11 @@ func GetRubberTree(ctx *pulumi.Context,
 
 // Input properties used for looking up and filtering RubberTree resources.
 type rubberTreeState struct {
-	Container *plant.Container `pulumi:"container"`
-	Diameter  *float64         `pulumi:"diameter"`
-	Farm      *string          `pulumi:"farm"`
-	Size      *string          `pulumi:"size"`
-	Type      *string          `pulumi:"type"`
+	Farm *string `pulumi:"farm"`
```

Comment:
I know this is an expected change, but can you explain it for my own edification? Thanks!

Reply:
Go was incorrectly populating the contents of resource state objects from the resource's properties. The generator has been fixed s.t. it uses the schema's `StateInputs`, which is consistent with all other SDKs (and the intent of the schema).